public/js/dom.js

```diff
@@ -0,0 +1,104 @@
+const resultContainer = document.querySelector(".result-container");
+
+const searchcontainer = document.querySelector(".search-result");
+const serchInput = document.getElementById("search");
+const reposDiv = document.createElement("div");
+
+reposDiv.classList.add("repos");
+resultContainer.appendChild(reposDiv);
+
+const delayKeyUp = (() => {
```

Comment:
I need to read the function to understand what it is used for, so a proper comment would help here explaining what is this and is it used for